examples/tools/google/bazeltoolchain/build_simple_bazel_project.rst

```diff
@@ -0,0 +1,184 @@
+.. _examples_tools_bazel_toolchain_build_simple_bazel_project:
+
+Build a simple Bazel project using Conan
```

Comment:
I mean in the "Read more section" of the tutorial, so that users interested in other build systems other than cmake can find the example there. The original idea is that there would be an example for all the supported build systems.

Reply:
Also, it would be good to link this example in other sections that talk about Bazel integration, like BazelToolchain, etc.

Reply:
Yeah, I just recalled to add also the integrations/bazel section